usaspending_api/settings.py

```diff
@@ -37,7 +37,8 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = False
 
-ALLOWED_HOSTS = ["*"]
+HOST = "localhost:3000"  # To be overwritten by Ansible
```

Comment:
The comment is unnecessary. Other values in this file are overwritten by the deploy process already.

Reply:
That sounds like the comment is insufficient, not unnecessary. This line of code would provide a reasonable developer with the assumption that HOST will be equal to that value. Our ops code breaks that assumption by changing the value behind the scenes (which I, for one, had no idea of before working on this ticket). That's a textbook reason to have a comment. 

I'm sure other values in this file operate the same way, and should I encounter them I would like to make similar comments about those as well. Our last dev "chapter" meeting did have a broker dev say you can never have enough comments.

Even if you find it unnecessary, does it do any harm?

Reply:
#1: there is nothing questionable, odd, or confusing about this variable. It has a clear name ("host") and is populated with the default hostname from django runserver.
#2: What occurs during a deploy doesn't alter the code or behavior of the variable, it simply alters the value. This is standard for many systems.
#3: There is no reason to specifically list Ansible, if the deploy process moves to a new tool this comment will most likely be forgotten and become outdated. So yes, it can "do harm"

> Our last dev "chapter" meeting did have a broker dev say you can never have enough comments.

This is obviously not true. It is terribly easy to have too many comments. I'll admit there is room for more "good comments" but a comment isn't guaranteed to provide value.